tests/lean/printStruct.lean

```diff
@@ -0,0 +1,28 @@
+/-!
+  Test #print command for structures and classes
+-/
+
+/- Structure -/
+#print Prod
```

Comment:
Could we update this test file to use `#guard_msgs`? It is the future. :-)

Reply:
Indeed it is! I'm very happy to update the tests!